openfe/tests/setup/atom_mapping/test_perses_scorers.py

```diff
@@ -0,0 +1,87 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import itertools
+import numpy as np
+from numpy.testing import assert_allclose, assert_
+import pytest
+from lomap import mcs as lomap_mcs
+from openfe.setup import perses_scorers, LigandAtomMapping
+
+pytest.importorskip('perses')
+try:
+    from perses.rjmc.atom_mapping import AtomMapper, AtomMapping
+except ImportError:
+    pass
```

Comment:
I can't remember 100% but I _think_ the importorskip might take precedence here? Do double check.